src/ResourceManager/Billing/Commands.Billing/help/Get-AzureRmEnrollmentAccount.md

```diff
@@ -0,0 +1,92 @@
+﻿---
+external help file: Microsoft.Azure.Commands.Billing.dll-Help.xml
+Module Name: AzureRM.Billing
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.billing/get-azurermenrollmentaccount
+schema: 2.0.0
+---
+
+# Get-AzureRmEnrollmentAccount
+
+## SYNOPSIS
+Get enrollment accounts.
+
+## SYNTAX
+
+### List (Default)
+```
+Get-AzureRmEnrollmentAccount [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### Single
+```
+Get-AzureRmEnrollmentAccount -Name <System.String>
```

Comment:
this is not accurate - the cmdlet has an ObjectId paraeter, but no name

Reply:
Fixed